schemas/paths/event-search.yaml

```
@@ -95,6 +95,106 @@ get:
       description: |
         Filter events previously tagged as suspicious via the [Update API](https://dev.fingerprint.com/reference/updateevent). 
         > Note: When using this parameter, only events with the `suspect` property explicitly set to `true` or `false` are returned. Events with undefined `suspect` property are left out of the response.
+    - name: vpn
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by VPN detection.  
+        > Note: When using this parameter, only events with the `vpn` property explicitly set to `true` or `false` are returned. Events with undefined `vpn` property are left out of the response.
```

The `event` object has no `vpn` property. We need to be more exact when describing how the filtering mechanism works. Please apply this pattern to all filter descriptions. 

```suggestion
        > Note: When using this parameter, only events with the `products.vpn.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.vpn` are left out of the response.
```